.vscode/settings.json

```diff
@@ -7,7 +7,21 @@
     "out": true // set this to false to include "out" folder in search results
   },
   "tslint.autoFixOnSave": true,
-  "editor.insertSpaces": true,
-  "editor.tabSize": 2,
-  "editor.detectIndentation": false
-}
\ No newline at end of file
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true,
+  "files.trimTrailingWhitespace": true,
```

Comment:
yes it can be removed, as I remember I added editorconfig file later, as setting tabs size in settings.json was incorrectly reformatting my global `settings.json`